core/css/public.scss

```
@@ -3,11 +3,11 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 #body-public {
-	--footer-height: calc(var(--default-line-height) + 2 * var(--default-grid-baseline));;
+	--footer-height: calc(var(--default-line-height) + 2 * var(--default-grid-baseline));
```

So instead you need to multiply the line height with the font size, or much easier:

```suggestion
	--footer-height: calc(1lh + 2 * var(--default-grid-baseline));
```